modules/core/05-port/types/module.go

```diff
@@ -81,6 +81,18 @@ type IBCModule interface {
 		channelID string,
 	) error
 
+	// TODO: consider removing timeout height and timeout timestamp added back for callbacks
+	OnSendPacket(
+		ctx sdk.Context,
+		portID string,
+		channelID string,
+		sequence uint64,
+		timeoutHeight clienttypes.Height,
+		timeoutTimestamp uint64,
```

Comment:
we should remove timeout info from the interface. Left for now since it breaks callbacks interface, but I don't see a reason to leave it for callbacks